TASK1/output.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+require 'form_validate.php';
+// Validate the form.
+// If errors are encountered during validation display them.
+if (!empty($errors)) {
+?>
+  <ul><?php foreach ($errors as $i) : { ?>
+        <li><?php echo $i; ?></li>
+    <?php
+
+        }
+      endforeach;
+    ?>
+  </ul>
+<?php
+}
+// When no issues are detected display the content.
+else {
+?>
+  <p>Hello <?= $_POST['firstname'] ?> <?= $_POST['lastname'] ?>
+  <p>
+    <img src=<?= $file_name ?> alt='User Image'>
+  <table class='marks'>
+    <thead>
+      <tr>
+        <th>Subjects</th>
+        <th>Score</th>
+      </tr>
+    </thead>
+    <tbody>
+      <?php
+
+      $subjects = explode("\n", $_POST['score']);
+      foreach ($subjects as $i) {
+        $subject = explode('|', $i);
+      ?>
+        <tr>
+          <td><?= $subject[0] ?></td>
```

Comment:
Our requirement was that user have to enter data only in the this format ;
Subject | Marks 
We need to validate that , currently we are allowing any kind of data to be submitted, for example if user enters 40 | English, or maybe some special characters like ``$%# | $#! ` we are still printing it which isn't right. Please update accordinly with proper validations